src/effects/effectmanifestparameter.h

```diff
@@ -202,21 +204,22 @@ class EffectManifestParameter {
     virtual const double& getDefault() const {
         return m_default;
     }
-    virtual void setDefault(const double& defaultValue) {
-        m_default = defaultValue;
-    }
 
     virtual const double& getMinimum() const {
         return m_minimum;
     }
-    virtual void setMinimum(const double& minimum) {
-        m_minimum = minimum;
-    }
 
     virtual const double& getMaximum() const {
```

Comment:
Can you remove the `virtual` keyword from these functions? There aren't any subclasses of EffectManifestParameter and I doubt there ever will be.